gradle/native-build.gradle

```diff
@@ -115,7 +220,7 @@ ext.aarBuild = { buildDir, projectDir, name, os ->
     }
 }
 
-ext.nativeBuild = { buildDir, projectDir, name, os ->
+ext.nativeBuildLib = { buildDir, projectDir, name, os ->
```

Comment:
Sorry, one more thing, the comments are outdated, since nativeBuildLib doesn't add the library to the jar:
```
println "Adding $n to native jar"
println "Adding $a to native jar"
println "Adding lib${name} to native jar"
```
Probably best to remove them.